doc/classes/CodeEdit.xml

```diff
@@ -500,20 +500,20 @@
 		<method name="unfold_all_lines">
 			<return type="void" />
 			<description>
-				Unfolds all lines, folded or not.
+				Unfolds all lines that are folded.
 			</description>
 		</method>
 		<method name="unfold_line">
 			<return type="void" />
 			<param index="0" name="line" type="int" />
 			<description>
-				Unfolds all lines that were previously folded.
+				Unfolds the line if it is folded or if it is hidden under a folded line.
 			</description>
 		</method>
 		<method name="unindent_lines">
 			<return type="void" />
 			<description>
-				Unindents selected lines, or in the case of no selection the caret line by one. Same as performing "ui_text_unindent" action.
+				Unindents all lines that are selected or have a caret on them. Uses spaces or a tab depending on [member indent_use_spaces]. Same as performing the [member ProjectSettings.input/ui_text_dedent] action. See [method indent_lines].
```

Comment:
```suggestion
				Unindents all selected lines or lines containing the caret. Uses spaces or a tab depending on [member indent_use_spaces]. Equivalent to performing the [member ProjectSettings.input/ui_text_dedent] action. See [method indent_lines].				
```

Reply:
For `lines that are selected or have a caret on them` I made this way to match other descriptions for consistency. 
Also in your suggestion, it would be `and` instead of `or`.